.travis.yml

```diff
@@ -0,0 +1,74 @@
+# Copyright 2018 Xaptum, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License
+
+sudo: required
+
+language: c
+
+services:
+  - docker
+
+compiler: gcc
+
+jobs:
+  include:
+
+# Release build, gcc
+  - env: TYPE=RELEASE
+    before_script:
+      - docker pull xaptum/xtt-cpp:0.1.0
+
+    script:
+      - docker run -v $(pwd):/xttcpp/ -d -t --name xttcpp xaptum/xtt-cpp:0.1.0
+      - docker exec -it xttcpp bash -c "cd xttcpp && mkdir -p build"
+      - docker exec -it xttcpp bash -c "cd xttcpp/build && cmake .. -DCMAKE_C_COMPILER=/usr/bin/gcc -DCMAKE_CXX_COMPILER=/usr/bin/g++"
+      - docker exec -it xttcpp bash -c "cd xttcpp/build && make"
+      - docker exec -it xttcpp bash -c "cd xttcpp/build && ctest -E tpm -VV"
```

Comment:
Just remove the `-E tpm` from the `ctest` line. I think that's a copy-paste from another project, probably `ecdaa`. But, it's un-needed here (heck, we don't even have any tests yet!)